plugins/agent-harness-flight-recorder/scripts/reporting.py

```
@@ -597,12 +577,49 @@ def inspect_episode(
         policy_version, policy_path
     )
 
+    if not locked:
+        projection = read_sealed_episode(root, policy_version, episode_id)
+        if (policy_version, episode_id) in load_forgotten(root):
+            raise VaultError("episode is forgotten for relationship policy")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)